corehq/apps/app_manager/id_strings.py

```diff
@@ -201,11 +201,28 @@ def mobile_ucr_annotation(module, uuid, annotation_index):
     )
 
 
+@pattern('m%d.enum.%s')
+def module_name_enum_variable(module, key_as_var):
+    return "m{module.id}.enum.{key_as_var}".format(
```

Comment:
are these going to be really long ids? 

tests below make it look like id == index and not the unique_id

Reply:
Not following the question. id is index, not the unique id, so these will be short.